metricflow/engine/models.py

```diff
@@ -47,20 +48,13 @@ def from_pydantic(cls, pydantic_metric: SemanticManifestMetric, dimensions: List
         )
 
     @property
-    def input_measures(self) -> List[MetricInputMeasure]:
+    def input_measures(self) -> Sequence[MetricInputMeasure]:
         """Return the complete list of input measure configurations for this metric."""
-        type_params = self.type_params
-
-        measures: List[MetricInputMeasure] = list(
-            type_params.measures if type_params is not None and type_params.measures is not None else []
+        assert self.type_params.input_measures, (
+            f"Metric {self.name} should have had input_measures populated by "
+            f"{AddInputMetricMeasuresRule.__class__.__name__}"
         )
-        if type_params.measure:
-            measures.append(type_params.measure)
```

Comment:
Did we remove this from DSI? Or is the input_measures property automatically wrapping this?

Reply:
Can you clarify what you mean by `remove this`?

Reply:
Sorry, I meant move this logic to DSI, I have no idea how that sentence came out that way.